manifests/pub_dir.pp

```diff
@@ -16,34 +16,8 @@
 
   ensure_packages('katello-client-bootstrap')
 
-  if $foreman_proxy_content::shared_with_foreman_vhost {
-    include foreman::config::apache
-
-    foreman::config::apache::fragment { 'pub_dir':
-      content     => template('foreman_proxy_content/httpd_pub.erb'),
-      ssl_content => template('foreman_proxy_content/httpd_pub.erb'),
-    }
-  } elsif $foreman_proxy_content::pulp {
-    pulp::apache::fragment { 'pub_dir':
-      ssl_content => template('foreman_proxy_content/httpd_pub.erb'),
-    }
-
-    file { "${apache::confd_dir}/pulp-vhosts80/pub_dir.conf":
-      ensure  => file,
-      content => template('foreman_proxy_content/httpd_pub.erb'),
-      owner   => 'apache',
-      group   => 'apache',
-      mode    => '0600',
-      notify  => Service['httpd'],
-    }
-  } else {
-    apache::vhost { 'foreman_proxy_content':
-      servername      => $servername,
```

Comment:
This class parameter is now unused. You can also drop `include apache` since pulpcore is responsible for that. `include foreman_proxy_content` is also redundant since no variable from it is used anymore.